liveblog.php

```diff
@@ -250,7 +251,11 @@ public static function init() {
 			 * Add liveblog support to the 'post' post type. This is done here so
 			 * we can possibly introduce this to other post types later.
 			 */
-			add_post_type_support( 'post', self::KEY );
+			$post_types = array( 'post' );
+			self::$supported_post_types = apply_filters( 'liveblog_modify_supported_post_types', $post_types );
```

Comment:
This filter should be [documented](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).

Reply:
I would rename it to `liveblog_supported_post_types`